backend/server/src/service/organisation.rs

```diff
@@ -0,0 +1,299 @@
+use anyhow::{bail, Result};
+use sqlx::{Pool, Postgres};
+use chrono::{DateTime, Local, NaiveDateTime, Utc};
+use serde::{Deserialize, Serialize};
+
+#[derive(Deserialize, Serialize)]
+pub struct OrganisationDetails {
+    pub id: i64,
+    pub name: String,
+    pub logo: Option<String>,
+    pub created_at: DateTime<Utc>
+}
+
+pub async fn get_organisation(id: i64, pool: Pool<Postgres>) -> Result<OrganisationDetails> {
+    let organisation_details = sqlx::query!
+    (
+        "
+            SELECT *
+                FROM organisations
+                WHERE id = $1
+        ",
+        id).fetch_optional(&pool)
+        .await?;
+
+    if let Some(result) = organisation_details {
+        let details = OrganisationDetails {
+            id: result.id,
+            name: result.name,
+            logo: result.logo,
+            created_at: result.created_at.and_utc()
+        };
+        return Ok(details);
+    }
+
+    bail!("error: failed to get organisation");
+}
+
+
+pub async fn update_organisation_logo(id: i64, logo: String, pool: Pool<Postgres>) -> Result<String> {
+    let dt = Local::now();
+    let current_time = dt.naive_utc();
+    let updated_organisation = sqlx::query!
+    (
+        "
+            UPDATE organisations
+                SET logo = $2, updated_at = $3
+                WHERE id = $1
+                RETURNING *
+        ",
+        id,
+        logo,
+        current_time
+    ).fetch_one(&pool)
+    .await;
+
+    match updated_organisation {
+        Ok(_) => Ok(logo),
+        Err(_) => bail!("error: Failed to update organisation logo")
+    }
+}
+
+pub async fn delete_organisation(organisation_id: i64, admin_id: i64, pool: Pool<Postgres>) -> Result<String, String> {
+   // Delete all organisation_admins with the corresponding organisation_id
+   let deleted_users = sqlx::query!
+    (
+        "
+            WITH user_exists AS (
```

Comment:
Only need to delete the Organisation, `CASCADE DELETE` will handle rest of rows.